tests/test_application_summary.py

```diff
@@ -42,23 +54,41 @@ def test_serialise_application_summary():
     assert applications[1].last_edited is None
 
 
-def test_dashboard_route(flask_test_client, requests_mock):
-    requests_mock.get(
-        "http://application_store/applications?account_id=test-user",
-        text=TEST_APPLICATION_STORE_DATA,
+def test_dashboard_route(flask_test_client, mocker):
+    mocker.patch(
+        "app.default.data.get_local_data",
+        return_value=TEST_APPLICATION_STORE_DATA,
     )
+
     response = flask_test_client.get(
         "/account/test-user", follow_redirects=True
     )
     assert response.status_code == 200
-    assert b"IN PROGRESS" in response.data
+    assert b"In Progress" in response.data
```

Comment:
Has sentence case been confirmed as final from the designs now? (for my own info!)

Reply:
yeah good point! afaik its the latest in the proto: https://fsd-funding-prototype.herokuapp.com/cof-demo/app-task-list-new